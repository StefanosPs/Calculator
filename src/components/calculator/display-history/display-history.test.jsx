import React from 'react';
import renderer from 'react-test-renderer';
 
import DisplayHistory from "./display-history";

it('renders NumberKey when there are no props', () => {
  const tree = renderer.create(<DisplayHistory />).toJSON();
  expect(tree).toMatchSnapshot(); 
});

it('renders NumberKey when there are props', () => {
    const tree = renderer.create(<DisplayHistory calculationHistory={[]}  />).toJSON();
    expect(tree).toMatchSnapshot();
});