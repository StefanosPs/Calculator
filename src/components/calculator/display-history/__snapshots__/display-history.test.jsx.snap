// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders NumberKey when there are no props 1`] = `
<div
  className="mt-2 bg-transparent border-0 shadow-sm mb-1 display-history container-fluid"
>
  
</div>
`;

exports[`renders NumberKey when there are props 1`] = `
<div
  className="mt-2 bg-transparent border-0 shadow-sm mb-1 display-history container-fluid"
/>
`;
